<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1778</width>
    <height>1187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFCFCFC;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background-color: transparent;	
    color:rgb(41, 128, 182);
    border:2px solid rgb(41, 128, 182);
    
	font: 75 10pt &quot;Arial&quot;;
}
QPushButton:hover{ 
color:rgb(53, 171, 239);
border-radius:6px;
border:2px solid #2980b6;
}

QPushButton:pressed{
border: 1px solid #3C3C3C;
border-radius:6px;
background:rgb(124, 195, 255);
}
#QRadioButton::indicator,
QCheckBox::indicator {
background-color: transparent;	
color:rgb(41, 128, 182);
border:2px solid rgb(41, 128, 182);
}
QCheckBox::indicator:unchecked:hover,
QCheckBox::indicator:checked:hover {
background-color: rgb(137, 195, 235);	
color:rgb(41, 128, 182);
border:2px solid rgb(41, 128, 182);
border-radius:3px
}
QCheckBox::indicator::checked {
background-color: rgb(137, 195, 235);	
color:rgb(41, 128, 182);
border:2px solid rgb(41, 128, 182);
border-radius:2px
}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="Maxmi">
          <property name="text">
           <string>区分大小写</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="whole_pair_check">
          <property name="text">
           <string>整词匹配</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="all_page">
          <property name="text">
           <string>对所有分页</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="only_match_check">
          <property name="text">
           <string>仅显示结果行</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="reset_button">
        <property name="text">
         <string>重 置 词 条</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="status_label">
        <property name="text">
         <string>等待上传中</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="in_word">
        <property name="styleSheet">
         <string notr="true">QTextEdit{
border:2px solid rgb(41, 128, 182);
font: 10pt &quot;Arial&quot;;
}
QScrollBar:vertical
{
       border: none;
       background: transparent;
       width: 6px;
}
 
QScrollBar::handle:vertical
{
       background-color: rgb(41, 128, 182);
       min-height: 30px;
	  border-radius:3px;
	  width: 6px;
   }
 
QScrollBar::add-line:vertical
{
       border: none;
       height: 0px;
 }
 
 QScrollBar::sub-line:vertical
{
       border: none;
       height: 0px;
}
 
 QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
 {	
	background:transparent;
      border-radius:3px;
}</string>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>键入保留关键字，多个关键字以换行区分</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="ex_word">
        <property name="styleSheet">
         <string notr="true">QTextEdit{
border:2px solid rgb(41, 128, 182);
font: 10pt &quot;Arial&quot;;
}
QScrollBar:vertical
{
       border: none;
       background: transparent;
       width: 6px;
}
 
QScrollBar::handle:vertical
{
       background-color: rgb(41, 128, 182);
       min-height: 30px;
	  border-radius:3px;
	  width: 6px;
   }
 
QScrollBar::add-line:vertical
{
       border: none;
       height: 0px;
 }
 
 QScrollBar::sub-line:vertical
{
       border: none;
       height: 0px;
}
 
 QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
 {	
	background:transparent;
      border-radius:3px;
}</string>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>键入排除关键字，多个关键字以换行区分</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="apply">
        <property name="text">
         <string>应 用 过 滤</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>368</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QTabWidget" name="tabs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>91</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QTabBar::tab{
background-color: #DEEBF7;
min-width:100px;
min-height:50px;
border-top-left-radius: 5px;
border-top-right-radius: 5px;
color: rgb(44, 62, 80);
font: Bold 9pt &quot;Arial&quot;;
border-top:2px solid rgb(41, 128, 182);
}
QTabBar::tab:hover{
background-color: rgb(137, 195, 235);
border:2px solid;
}
//设置内容区域边框
QTabWidget::pane{
border:2px solid rgb(41, 128, 182);
background-color:#FFFFFF	
}
//标题栏左侧间距
QTabWidget::tab-bar {
    left: 1px;
}</string>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QFrame" name="search_info">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1778</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="menu_open"/>
    <addaction name="separator"/>
    <addaction name="norm_input"/>
    <addaction name="separator"/>
    <addaction name="menu_download"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionUpLoadFile">
   <property name="text">
    <string>UpLoadFile</string>
   </property>
  </action>
  <action name="menu_open">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="statusTip">
    <string>打开一个log文件</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
  </action>
  <action name="menu_batchopen">
   <property name="text">
    <string>批量打开</string>
   </property>
  </action>
  <action name="menu_locraw">
   <property name="text">
    <string>定位到行</string>
   </property>
  </action>
  <action name="menu_download">
   <property name="text">
    <string>下载</string>
   </property>
   <property name="statusTip">
    <string>下载词条过滤结果和过滤条件信息</string>
   </property>
  </action>
  <action name="action1_4">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action1_5">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="norm_input">
   <property name="text">
    <string>扩展正则过滤</string>
   </property>
   <property name="statusTip">
    <string>使用正则表达式进行词条过滤（仅支持全文过滤）</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
